packages/web-components/src/components/cbp-tooltip/cbp-tooltip.specs.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Tooltip/Specifications" />
+
```

Comment:
Specs need to be cleaned up and fleshed out. Use proper capitalization and punctuation. 

The purpose should probably be discussed for specific wording with clarifying guidance from Jeremy.

The accessibility section especially should reference the role, tabindex, keyboard nav, etc.

Reply:
update doc based on UX spec details & reviewed for capitalization & punctuation